### PR TITLE
[ReferentSpace] allow contact candidates of department

### PR DESCRIPTION
**Pull request by ottaviano**
[ch9126]

### CODE REVIEW

File: src/Mailchimp/Campaign/SegmentConditionBuilder/ReferentToCandidateConditionBuilder.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        }
+
+        $conditions[] = [
+            'condition_type' => 'TextMerge',
+            'op' => 'contains',
+            'field' => MemberRequest::MERGE_FIELD_REFERENT_TAGS,
+            'value' => $campaign->getLabel(),
+        ];
+
+        return $conditions;
```
Comment (Remg) [inline src/Mailchimp/Campaign/SegmentConditionBuilder/ReferentToCandidateConditionBuilder.php]: maybe you can directly return the array here

File: src/Mailchimp/Campaign/SegmentConditionBuilder/MunicipalChiefToAdherentConditionBuilder.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        $conditions[] = [
+            'condition_type' => 'TextMerge',
+            'op' => 'starts',
+            'field' => MemberRequest::MERGE_FIELD_CITY,
+            'value' => $cityName.' (',
+        ];
+
+        return $conditions;
```
Comment (Remg) [inline src/Mailchimp/Campaign/SegmentConditionBuilder/MunicipalChiefToAdherentConditionBuilder.php]: maybe you can directly return the array here

File: src/Mailchimp/Campaign/SegmentConditionBuilder/ReferentToCandidateConditionBuilder.php (hunk 1 above)
Comment (ottaviano) [reply src/Mailchimp/Campaign/SegmentConditionBuilder/ReferentToCandidateConditionBuilder.php]: done 👍

File: src/Mailchimp/Campaign/SegmentConditionBuilder/MunicipalChiefToAdherentConditionBuilder.php (hunk 2 above)
Comment (ottaviano) [reply src/Mailchimp/Campaign/SegmentConditionBuilder/MunicipalChiefToAdherentConditionBuilder.php]: done 👍

**Pull request merged**